src/client/Microsoft.Identity.Client/ApiConfig/WebView2Options.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.Internal.Logger;
+using Microsoft.Identity.Client.PlatformsCommon.Factories;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Options for using the modern Windows embedded browser WebView2. 
+    /// For more details see https://aka.ms/msal-net-webview2
+    /// </summary>
+#if !SUPPORTS_WEBVIEW2
+    [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
+#endif
+    public class WebView2Options
+    {
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        public WebView2Options()
+        {
+            ValidatePlatformAvailability();
+        }
+
+        /// <summary>
+        /// Forces a static title to be set on the window hosting the browser. If not configured, the widow's title is set to the web page title.
```

Comment:
What is it for? is it for branding the application?
What is the title of the web page? does it vary as a function of the action dialog happening with Azure AD? 

In general for branding the best is to do it at the application registration level: https://docs.microsoft.com/azure/active-directory/develop/howto-add-branding-in-azure-ad-apps

Reply:
It is less for branding of the application and more about having less confusion around where this window comes from. End-users report being confused about it. It has been asked several times from the mobile front as well.

Especially useful for GCM which shares the client_id of VS (for many reasons) and cannot easily move off of it.